src/commands/apply/datacenter.ts

```diff
@@ -82,7 +53,10 @@ async function apply_datacenter_action(options: ApplyDatacenterOptions, name: st
   const envs = Deno.env.toObject();
   for (const key of Object.keys(envs)) {
     if (key.startsWith('ARC_')) {
-      flag_vars[key.replace('ARC_', '')] = envs[key];
+      const key_var = key.replace('ARC_', '');
```

Comment:
FYI this change is because if you set an env variable, it would override manually set vars which 1. was confusing the hell out of me because I had `ARC_region` set from previous testing and 2. is backwards when a manually passed in flag is overridden by the environment